### PR TITLE
Fixes #28 Consistent class prefixes of XApi

### LINKED ISSUE #28: Inconsistent use of naming in the xapi java files
xapi files are prefixed with a variety of variants of xapi.

Xapi
XApi
XAPI

The most common one seems to be XApi. These should be standardised.

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

File: src/main/java/org/apereo/openlrs/controllers/xapi/XAPIExceptionHandlerAdvice.java [hunk 1]
```diff
@@ -49,18 +49,18 @@
  * @author Gary Roybal, groybal@unicon.net
  */
 @ControllerAdvice(annotations = RestController.class)
-public class XAPIExceptionHandlerAdvice {
+public class XApiExceptionHandlerAdvice {
```
Comment (relong76) [inline src/main/java/org/apereo/openlrs/controllers/xapi/XAPIExceptionHandlerAdvice.java]: You'll need to change the file names for these class name changes as well.

File: src/main/java/org/apereo/openlrs/controllers/xapi/XAPIHeaderFilter.java [hunk 2]
```diff
@@ -32,7 +32,7 @@
  *
  */
 @Component
-public class XAPIHeaderFilter extends OncePerRequestFilter {
+public class XApiHeaderFilter extends OncePerRequestFilter {
```
Comment (relong76) [inline src/main/java/org/apereo/openlrs/controllers/xapi/XAPIHeaderFilter.java]: You'll need to change the file names for these class name changes as well.

File: src/main/java/org/apereo/openlrs/controllers/xapi/XAPIRequestValidationFilter.java [hunk 3]
```diff
@@ -32,9 +32,9 @@
  *
  */
 @Component
-public class XAPIRequestValidationFilter extends OncePerRequestFilter {
+public class XApiRequestValidationFilter extends OncePerRequestFilter {
```
Comment (relong76) [inline src/main/java/org/apereo/openlrs/controllers/xapi/XAPIRequestValidationFilter.java]: You'll need to change the file names for these class name changes as well.

File: src/main/java/org/apereo/openlrs/controllers/xapi/XapiApiController.java [hunk 4]
```diff
@@ -44,12 +44,12 @@
  */
 @RestController
 @RequestMapping("/v1/xAPI/statements")
-public class XapiApiController {
-  private final Logger logger = LoggerFactory.getLogger(XapiApiController.class);
+public class XApiApiController {
```
Comment (relong76) [inline src/main/java/org/apereo/openlrs/controllers/xapi/XapiApiController.java]: You'll need to change the file names for these class name changes as well.

File: src/main/java/org/apereo/openlrs/model/xapi/XAPIErrorInfo.java [hunk 5]
```diff
@@ -31,7 +31,7 @@
  * Class used to hold info for a XAPI-call-related error.  This object will be JSONified and returned as the XAPI call response.
  * @author Gary Roybal, groybal@unicon.net
  */
-public class XAPIErrorInfo {
+public class XApiErrorInfo {
```
Comment (relong76) [inline src/main/java/org/apereo/openlrs/model/xapi/XAPIErrorInfo.java]: You'll need to change the file names for these class name changes as well.

File: src/main/java/org/apereo/openlrs/controllers/xapi/XAPIExceptionHandlerAdvice.java (hunk 1 above)
Comment (adrianfish) [inline src/main/java/org/apereo/openlrs/controllers/xapi/XAPIExceptionHandlerAdvice.java]: I'm having some git issues with case. New one to me :( I was just about to close it. I'll reissue later.

**Pull request closed without merging**